src/SASjsApiClient.ts

```diff
@@ -62,8 +62,8 @@ export class SASjsApiClient {
 
   /**
    * Executes code on a SASJS server.
-   * @param serverUrl - a server url to execute code.
    * @param code - a string of code to execute.
+   * @param authConfig - an object for authentication.
```

Comment:
Can we get more information on this?  How to set up an authConfig?  What about an example?